addons/website/static/tests/tours/snippet_table_of_content.js

```diff
@@ -59,7 +43,7 @@ registerWebsitePreviewTour('snippet_table_of_content', {
     scrollToHeading(2),
     checkTOCNavBar(1, 0),
     scrollToHeading(3),
-    checkTOCNavBar(1, 1),
+    checkTOCNavBar(1, 0),
```

Comment:
why we need this change?

Reply:
I guess since i missed the above step, the scroll behaviour was casuing `checkTOCNavBar(1, 1),` to break the step, let me check that and update it.

Reply:
@msh-odoo it was not the issue of the step i missed , it is happening because when content scrollToHeading(3) the active index for the link in that time is not the 1st one rather it scrolls to 0th hence i think we can, it could be related to the way `this.anchor.scrollIntoView` behaves. I think we can move forward with this change.

Reply:
@shsa-odoo I found it strange because we called ```scrollToHeading``` before and it is working fine there.

Reply:
@msh-odoo Its strange indeed. What do you suggest should be done here if the focus is set to 0th making it active instead of the 1st while scrolling to the `Heading(3)`.